config/routes.rb

```diff
@@ -5,8 +5,10 @@
 	mount Sidekiq::Web => '/sidekiq'
 	
 	mount_devise_token_auth_for 'User', at: 'auth'
-	resources :books, only: [:index, :show]
+	resources :books, only: [:index, :show, :external_api_get]
 	resources :rents, only: [:index, :create]
 	resources :book_suggestions, only: [:create]
 
+	get 'open_library/:isbn', to: 'books#external_api_get', as: :external_api_get
```

Comment:
🤔 
As i said before, this is a very general-name route for this service. Even thougth, we dont have a restriction on the name of the path, a better path name would be `open_library_book` for example